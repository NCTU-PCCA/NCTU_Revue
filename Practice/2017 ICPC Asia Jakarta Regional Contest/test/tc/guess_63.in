25204333592 35
<<<><>>>>><<<>><<<><>><>><<<<><<<>>
