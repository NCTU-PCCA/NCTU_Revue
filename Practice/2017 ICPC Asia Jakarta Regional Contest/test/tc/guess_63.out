25204333592 25204333591 25204333590 1 25204333589 2 3 4 5 6 25204333588 25204333587 25204333586 7 8 25204333585 25204333584 25204333583 9 25204333582 10 11 25204333581 12 13 25204333580 25204333579 25204333578 25204333577 14 25204333576 25204333575 25204333574 15 16
